es-core/src/components/ImageGridComponent.h

```diff
@@ -344,26 +350,69 @@ void ImageGridComponent<T>::updateTiles()
 	if (!mTiles.size())
 		return;
 
-	int img = getStartPosition();
-
-	for(int ti = 0; ti < mTiles.size(); ti++)
+	// Stop updating the tiles at highest scroll speed
```

Comment:
The other thing I noticed is that when you get to the fast scroll, the grid completely disappears while the letter overlay is rendered.  I understand we don't want to load the actually images, but could we just render the tiles with the default texture?

Reply:
We can, it's a design choice I made. I didn't like how it looked with default texture so I made it completely fade out like metadata do. But I'm still open to discussion if you find that change necessary.

Reply:
For now we can just leave it as is and then get some feedback from the wider user base once it is enabled.